.github/workflows/generate-sdks.yml

```diff
@@ -10,36 +10,13 @@ on:
   workflow_dispatch:
 
 jobs:
-  run:
-    runs-on: ubuntu-latest
+  open-api-workflow-generate-sdks:
+    uses: maykinmedia/open-api-workflows/.github/workflows/generate-sdks.yml@initial-setup
```

Comment:
as @Coperh mentioned as well, I think it would be good to update the main branch of `maykinmedia/open-api-workflows` as well

Reply:
Let's also use versions for the actions, so we will pin their versions 
And it it necessary to use the whole path? 
Can we have something similar to `maykinmedia/setup-django-backend@v1.3` ?

Reply:
I thought the ideas was to change it in only one place so it is the same across the projects

Reply:
Regarding the main branch of open-api-workflows, I'll merge `initial-setup` into that whenever the PR's across all projects are ready to be merged.

In this situation the path is required, using a workflow from a external repository, see [documentation](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_iduses).